src/actions.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { isNgrxFormsAction } from './actions';
+
+describe('action', () => {
+  it('should return true if type is ngrx/forms/', () => {
+    expect(isNgrxFormsAction({ type: 'ngrx/forms/' })).toBeTruthy();
```

Comment:
You expect the return value to be `true`, not just any truthy value, therefore you should assert on the concrete value (same for the other tests).